# target_frame: base_link # Leave disabled to output scan in pointcloud frame
transform_tolerance: 0.01
min_height: 0.2
max_height: 1.6

angle_min: -3.141592            # -M_PI/2
angle_max: 3.141592             # M_PI/2
angle_increment: 0.0034906584 # 2.0 * M_PI / 1800 (at 10 Hz)
scan_time: 0.1                # 10 Hz
time_increment: 5.5556e-5     # 0.1 / 1800
range_min: 0.1
range_max: 150.0
use_inf: true
inf_epsilon: 1.0

# Concurrency level, affects number of pointclouds queued for processing and number of threads used
# 0 : Detect number of cores
# 1 : Single threaded
# 2->inf : Parallelism level
concurrency_level: 0